public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -977,6 +977,7 @@ export const Explorer = ({
                   endTime={endTime}
                   setStartTime={setStartTime}
                   setEndTime={setEndTime}
+                  notifications={notifications}
```

Comment:
Let's use `coreRefs.notifications`directly in direct_search.tsx instead of passing it through props.

Reply:
made this change